2025/social-wg.html

```diff
@@ -319,8 +319,7 @@ <h2>Success Criteria</h2>
         </p>
 
         <p>
-      Each substantive change should contain a section detailing all known security and privacy implications
-      for implementers, authors, and end users.
+      Each specification should contain separate sections detailing all known security and privacy implications for implementers, authors, and end users.
```

Comment:
```suggestion
      Each specification should contain separate sections detailing all known security and privacy implications for implementers, authors, and users.
```

drop "end". The Web Platform Design Principles just uses "users" per https://www.w3.org/TR/design-principles/#priority-of-constituencies